### PR TITLE
Move all srcdata files to new repo

### PR DESCRIPTION
**Pull request by HighwayofLife**

### CODE REVIEW

**Comment by stiangre** [comment]
There are three things I don't like about this approach:
1. We lose the revision history for all the files a second time (unlike the code, the revision history was not maintained when the files were moved from Subversion to Git). I would like us to recreate the entire revision history when creating a new repo.
2. The master for the other repositories are all on SourceForge. Please note that it's not possible to move all the repositories over to Github since we have multi-GB-sized files for 3D projects (and these are too big for Github).
3. I don't think we need git-lfs for simple picture files ... even when binary. We are probably better off just having a normal (separate) repository for these files.

To summarize: I think we should create a new git repo on SF.net, and use the freecol-bot for mirroring to Github just like we do for our main repository. This repository should be created using a rewritten history from the Old-SVN and current Git repo.
